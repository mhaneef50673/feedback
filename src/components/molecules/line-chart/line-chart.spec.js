import React from 'react';
import { mount } from 'enzyme';
import { mountToJson } from 'enzyme-to-json';
import LineChart from './index';
import {
  defaultChartOptions,
  chartSeries
} from '../feedback-trend/test.data';

describe('LineChart Component', () => {
  it('should render the component', () => {
    const tree = mount(
      <LineChart
        chartOptions={defaultChartOptions}
        series={chartSeries}
      />
    )
    expect(mountToJson(tree)).toMatchSnapshot();
  });
});