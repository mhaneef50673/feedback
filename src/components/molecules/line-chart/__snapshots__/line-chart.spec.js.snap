// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LineChart Component should render the component 1`] = `
<LineChart
  chartOptions={
    Object {
      "chart": Object {
        "height": 350,
        "type": "line",
        "zoom": Object {
          "enabled": false,
        },
      },
      "colors": Array [
        "#c60c30",
      ],
      "dataLabels": Object {
        "enabled": false,
      },
      "grid": Object {
        "row": Object {
          "colors": Array [
            "#f3f3f3",
            "transparent",
          ],
          "opacity": 0.5,
        },
      },
      "stroke": Object {
        "curve": "straight",
      },
      "title": Object {
        "align": "left",
        "text": "Feedback trend by date",
      },
      "xaxis": Object {
        "categories": Array [],
        "convertedCatToNumeric": true,
        "title": Object {
          "text": "Date",
        },
      },
      "yaxis": Object {
        "labels": Object {
          "formatter": [Function],
        },
        "min": 0,
        "tickAmount": 5,
        "title": Object {
          "text": "count",
        },
      },
    }
  }
  series={
    Array [
      Object {
        "data": Array [],
        "name": "Feedbacks received",
      },
    ]
  }
>
  <r
    height={350}
    options={
      Object {
        "chart": Object {
          "height": 350,
          "type": "line",
          "zoom": Object {
            "enabled": false,
          },
        },
        "colors": Array [
          "#c60c30",
        ],
        "dataLabels": Object {
          "enabled": false,
        },
        "grid": Object {
          "row": Object {
            "colors": Array [
              "#f3f3f3",
              "transparent",
            ],
            "opacity": 0.5,
          },
        },
        "stroke": Object {
          "curve": "straight",
        },
        "title": Object {
          "align": "left",
          "text": "Feedback trend by date",
        },
        "xaxis": Object {
          "categories": Array [],
          "convertedCatToNumeric": true,
          "title": Object {
            "text": "Date",
          },
        },
        "yaxis": Object {
          "labels": Object {
            "formatter": [Function],
          },
          "min": 0,
          "tickAmount": 5,
          "title": Object {
            "text": "count",
          },
        },
      }
    }
    series={
      Array [
        Object {
          "data": Array [],
          "name": "Feedbacks received",
        },
      ]
    }
    type="line"
    width="100%"
  >
    <div
      height={350}
      options={
        Object {
          "chart": Object {
            "height": 350,
            "type": "line",
            "zoom": Object {
              "enabled": false,
            },
          },
          "colors": Array [
            "#c60c30",
          ],
          "dataLabels": Object {
            "enabled": false,
          },
          "grid": Object {
            "row": Object {
              "colors": Array [
                "#f3f3f3",
                "transparent",
              ],
              "opacity": 0.5,
            },
          },
          "stroke": Object {
            "curve": "straight",
          },
          "title": Object {
            "align": "left",
            "text": "Feedback trend by date",
          },
          "xaxis": Object {
            "categories": Array [],
            "convertedCatToNumeric": true,
            "title": Object {
              "text": "Date",
            },
          },
          "yaxis": Object {
            "labels": Object {
              "formatter": [Function],
            },
            "min": 0,
            "tickAmount": 5,
            "title": Object {
              "text": "count",
            },
          },
        }
      }
      series={
        Array [
          Object {
            "data": Array [],
            "name": "Feedbacks received",
          },
        ]
      }
      type="line"
      width="100%"
    />
  </r>
</LineChart>
`;
